docs/UserGuide.md

```diff
@@ -629,22 +680,22 @@ Format: `list`
 
 ### Finding patients: `find`
 
-You can find specific patients using the `find` command.
+Finds patients whose parameters contain the given keywords.
```

Comment:
I think this reverts a change I suggested (and you fixed accordingly) in PR #376 about using second-person language.

Reply:
Whoops, not sure why this happened, maybe `CTRL+Z`ed too many times?